rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/NodeSource.java

```diff
@@ -392,11 +392,16 @@ public BooleanWrapper acquireNode(String nodeUrl, Client provider) {
             ((AbstractRMNode) rmNode).copyLockStatusFrom(deployingNode);
         }
 
-        //we notify the configuration of the node to the rmcore
-        //it then will be seen as "configuring"
-        rmcore.internalRegisterConfiguringNode(rmNode);
+        boolean isNodeAdded = rmcore.registerAvailableNode(rmNode).getBooleanValue();
 
-        return new BooleanWrapper(true);
+        if (isNodeAdded) {
+            // if the node is successfully added we can let it configure
```

Comment:
```
if(isNodeAdded){
// asynchronously. It will then be seen as "configuring"
rmcore.internalRegisterConfiguringNode(rmNode);
}
return new BooleanWrapper(isNodeAdded)
```